application/actions/watches.go

```diff
@@ -0,0 +1,91 @@
+package actions
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/buffalo/render"
+	"github.com/gobuffalo/pop/v5"
+
+	"github.com/silinternational/wecarry-api/api"
+	"github.com/silinternational/wecarry-api/models"
+)
+
+// swagger:operation GET /watches Watches UsersWatches
+//
+// List the User's Watches
+//
+// ---
+// responses:
+//   '200':
+//     description: A list of the user's watches
+//     schema:
+//       "$ref": "#/definitions/Watches"
+func watchesMine(c buffalo.Context) error {
+	cUser := models.CurrentUser(c)
+	tx := models.Tx(c)
+
+	watches := models.Watches{}
+	if err := watches.FindByUser(tx, cUser, "Owner", "Destination", "Origin"); err != nil {
+		return reportError(c, &api.AppError{
+			HttpStatus: http.StatusInternalServerError,
+			Key:        api.WatchesLoadFailure,
+			Err:        err,
+		})
+	}
+
+	var output api.Watches
+	output, err := convertWatchesToAPIType(c, tx, watches, cUser)
+	if err != nil {
+		return reportError(c, appErrorFromErr(err))
+	}
+
+	return c.Render(200, render.JSON(output))
+}
+
+func convertWatchesToAPIType(ctx context.Context, tx *pop.Connection, watches models.Watches, user models.User) (api.Watches, error) {
+	var output api.Watches
+
+	if err := api.ConvertToOtherType(watches, &output); err != nil {
```

Comment:
Would it be too inefficient to convert one watch at a time in a function called `convertWatchToAPIType`? That way, if only one Watch is needed in a later endpoint, you would have consistency by calling the same function. This was done in organizations.go

Reply:
:+1: